units/XNL0403/XNL0403_Script.lua

```diff
@@ -112,17 +120,19 @@ XNL0403 = Class(NExperimentalHoverLandUnit, SlowHover) {
         end
     end,
 
+    ---@param self XNL0403
     DestroyMovementSmokeEffects = function(self)
         self.SmokeEmitters:Destroy()
     end,
 
--- DYING =====================
-
+    --- This is a WOF event which is in here cause I like WOF! Sorry, I can't be crushed during sinking!
```

Comment:
What is WOF?

Reply:
@Exotic-Retard are you able to elaborate?